bindata/network/kuryr/005-daemon.yaml

```diff
@@ -75,6 +75,9 @@ spec:
             port: {{ default 8090 .DaemonProbesPort }}
           initialDelaySeconds: 15
 {{ end }}
+        ports:
+        - name: metrics-port
+          containerPort: 9103
```

Comment:
Damn, I'm pretty sure one day it'll get taken by another service and we'll never know until we deploy, because we don't have any CI with Kuryr.

Reply:
I will add a line for Kuryr in [here](https://github.com/prometheus/prometheus/wiki/Default-port-allocations) with a new port, then we will be safe.

Reply:
Ha, nice. :D So that one's already taken.

9409 and 9410 are taken by ovn-kubernetes, so I bet we can reserve a port too.

Reply:
I'll allocate other ports on that wiki and here when another round of review is taken here. As stated in the wiki, we must only allocate when ready to share with others.